posts/apprenticeship.md

```diff
@@ -1,28 +1,32 @@
-# Introduction:
-
-Awe and Reverence (A&R) is a software company dedicated to build a God-glorifying, for-profit (read: sustainable), for-Kingdom benefit, cutting-edge Christian software company.
-
-In pursuit of that, our founder ([Jonathan Tsai](https://www.linkedin.com/in/jontsai/)) started an apprenticeship program at A&R to train aspiring Christian software engineers to equip them with tangible skills to succeed in their craft.  In the SF Bay Area, there is an ever-increasing number of people coming out of coding bootcamps trying to transition into software roles from other industries, but unfortunately, that train has left about 3-4 years ago (circa 2015) where companies might give these bootcamp grads a chance; many people are now struggling to find employment after these bootcamps.
-
-Jonathan has mentored dozens of engineers over the years, and through the Maskil apprenticeship has provided a post-bootcamp onboarding experience for more junior engineers to get real world, collaborative, team experience. Several of them have gone on to full-time software engineering roles and are junior, mid-level, and senior-level engineers at household-name tech companies.
-
-Not only will you gain invaluable experience, you'll also be paid for your work! Each feature/task will have a bounty with a dollar amount associated with it. The larger the feature/task, the larger the bounty. We don't believe in muzzling the ox here.
-
-# Qualifications:
-
-Born again Christian that is FAT!
-
-**F** - Faithful  
-**A** - Available  
-**T** - Teachable  
-
-# Responsibilities: 
-
-The first project that A&R has  been focused on is called Maskil; envisioned to be the FIRST EVER digital hymnal and worship study Bible. (Imagine every worship song, every language, with (Rap)Genius-style inline annotations and two-way links to Bible passages). Currently Maskil has over 1000 songs in English, 650+ songs in Korean, and growing!.
-
-We have a feature roadmap planned out for years. You will be working collaboratively with a small team of engineers. Some of the technologies you will learn and be exposed to include: 
-- Django
-- React
-- MySQL
-- LESS CSS
-- Git Workflow
+---
```

Comment:
I think this should just be a regular page under `pages/apprenticeship.md` -- what do you think?

Reply:
This was a file that was already in the code. We just used it as a test page. We can put the posts in the pages directory if you would prefer that.

Reply:
Yes, please. I must have missed it during the original implementation.

However, if it breaks the Markdown rendering, we can hold off and save it for a separate PR.

Actually, now that I look at it, it's somewhat odd in that we have two files:
- https://github.com/aweandreverence/www.aweandreverence.com/blob/master/pages/apprenticeship.js
- https://github.com/aweandreverence/www.aweandreverence.com/blob/master/pages/posts/apprenticeship.md

It seems that only one of them is being used.

I think we should just delete `apprenticeship.md`

Reply:
Sounds good. I'll do that tonight.